charmcraft/application/commands/store.py

```diff
@@ -1992,9 +1992,10 @@ def run(self, parsed_args: argparse.Namespace) -> int:
             if const.STORE_REGISTRY_ENV_VAR in os.environ:
                 # If the user has specified a registry to use, replace what the store
                 # gives with that registry.
-                registry_url = os.environ[const.STORE_REGISTRY_ENV_VAR][7:]
+                registry_url = os.environ[const.STORE_REGISTRY_ENV_VAR]
+                registry_url_without_scheme = registry_url.partition("://")[2]
```

Comment:
Is there a particular reason for not using `urlparse` for that ?

Reply:
`urlparse` doesn't have an easy way to present just the sections I need, so I'd still have to do some manual string manipulation after parsing it.